### ISSUE TITLE
Fine-tuning model configuration 

### ISSUE THREAD

**Issue by XZhang97666** [justinphan3110/SciFive#15]
I want to double check that the HOC task fine-tuned with learning rate=0.01 and step=45000 (show in the notebook), batch size=64 for base model. The last step checkpoint is used to calculate the F1 and other evaluation metric for the test set. This is correct? Please let me know if I miss some other details.

**Comment by justinphan3110** [comment]
Hi @XZhang97666 , we have just updated our code to use with the [T5X](https://github.com/google-research/t5x). You can have a look at it here: [finetunning_biot5x_blurb.ipynb](https://github.com/justinphan3110/SciFive/blob/main/biot5x/examples/finetunning_biot5x_blurb.ipynb). 

Note that for the HoC metric, we use the [Micro_F1](https://github.com/michiyasunaga/LinkBERT/blob/main/src/seqcls/utils_hoc.py#L230) to be comparable with the [BLURB Leaderboard](https://microsoft.github.io/BLURB/leaderboard.html).

**Comment by zoeyhl** [comment]
Hi, how are precision and recall calculated for HOC? are they also averaged by "micro" or "samples"?
